aalto/jupyterhub.rst

```diff
@@ -285,7 +285,8 @@ to give Github all our information, so you can also send tickets to
 CS-IT.
 
 Students and others who have difficulty in usage outside of a course
-can contact CS-IT via the guru alias.
+can contact CS-IT via the :doc:`guru <jupyterhub-instructors#more-info>`
```

Comment:
Didn't have a test environment available, haven't tested if this is the correct syntax